x11-packages/kf6-baloo/build.sh

```diff
@@ -0,0 +1,22 @@
+TERMUX_PKG_HOMEPAGE="https://invent.kde.org/frameworks/baloo"
+TERMUX_PKG_DESCRIPTION="A framework for searching and managing metadata"
+TERMUX_PKG_LICENSE="LGPL-2.0-only, LGPL-3.0-only"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="6.21.0"
+_KF6_MINOR_VERSION="${TERMUX_PKG_VERSION%.*}"
+TERMUX_PKG_SRCURL="https://download.kde.org/stable/frameworks/${_KF6_MINOR_VERSION}/baloo-${TERMUX_PKG_VERSION}.tar.xz"
```

Comment:
I'd prefer not adding a global helper variable for this.
```suggestion
TERMUX_PKG_SRCURL="https://download.kde.org/stable/frameworks/${TERMUX_PKG_VERSION%.*}/baloo-${TERMUX_PKG_VERSION}.tar.xz"
```
Especially when it's only that one instance where it ends up getting used.